src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -28,6 +28,9 @@ type Query {
     - [HTTP resolver](#http-resolver): call an HTTP endpoint upon a query or mutation
     - [VTL resolver](#vtl-resolver) (most advanced): use VTL mapping templates to customize the query and mutation logic
 
+3. Secure your custom query or mutation with [field-level authorization rules](/cli/graphql/authorization-rules/)]
+    - Note: Only API Key, IAM, custom, and static group authorization rules are supported on a custom query or mutation.
```

Comment:
So what is not supported? Dynamic auth? If that's the known thing that is unsupported it would be better to explicitly call that out.

Reply:
Updated